templates/asp-app.json

```diff
@@ -95,16 +95,101 @@
     },
     "java_container_version": {
       "type": "string"
+    },
+    "storageAccountSkuName": {
+      "type": "string",
+      "defaultValue": "Standard_LRS",
+      "metadata": {
+        "description": "The name of the App Service Plan."
```

Comment:
this doesn't seem right?

Reply:
changed the description to : The name of replciation strategy - LRS Locally-redundant storage.